llvm/cmake/modules/LLVMExternalProjectUtils.cmake

```diff
@@ -56,11 +56,13 @@ endfunction()
 #     Use provided strip tool instead of the default one.
 #   TARGET_TRIPLE triple
 #     Optional target triple to pass to the compiler
+#   FOLDER
+#     For IDEs, the Folder to put the targets into.=
```

Comment:
Maybe if this is only affecting IDEs, then perhaps the property's name should reflect this, i.e., `IDE_FOLDER`? Unfortunately, directories are also called "folders" at least on Windows, so I see a slim chance of misunderstanding this as the path from which the external project is loaded.

Reply:
It's already called FOLDER in https://github.com/llvm/llvm-project/blob/main/llvm/cmake/modules/AddLLVM.cmake#L2080-L2083. I would prefer to keep it consistent.